CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 2.2.3
```

Comment:
The version number isn't correct. This will be release in `3.0.0`. Merging it and updating this to `master` to denote all changes currently in master.